nupic/research/frameworks/vernon/experiments/meta_cl_experiment.py

```diff
@@ -118,6 +123,12 @@ def setup_experiment(self, config):
         self.replay_batch_size = config.get("replay_batch_size", 64)
         self.num_fast_steps = config.get("num_fast_steps", 1)
 
+        assert "fast_params" in config
+        fast_named_params = filter_params(self.model,
+                                          include_patterns=config["fast_params"])
+        self.fast_param_names = list(fast_named_params.keys())
```

Comment:
Can't we just return the keys directly, since the params are not being used at all?

Reply:
We could, but the filter_params is intended to be more general. We use just the names, but it could be helpful to retrieve the params to reinitialize them, for instance.

Reply:
But are we using it for that? It sounds a bit like overengineering for some future requirement but may not even happen.